src/main/java/com/google/opensesame/servlets/PersonObject.java

```diff
@@ -1,38 +1,81 @@
 package com.google.opensesame.servlets;
 
+import com.google.opensesame.github.GitHubGetter;
+import java.io.IOException;
 import java.util.ArrayList;
+import org.kohsuke.github.GHUser;
+import org.kohsuke.github.GitHub;
 
+/** A class containing basic user data. This is intended to be serialized and sent to the client. */
 public class PersonObject {
-  public static String ENTITY_NAME = "User";
-  public static String GITHUB_ID_FIELD = "github-id";
-  public static String TAG_LIST_FIELD = "tags";
-
-  private String name;
-  private String description;
-  private ArrayList<String> interestTags;
-  private String gitHubID;
-
-  public PersonObject(
-      String name, String gitHubID, String description, ArrayList<String> interestTags) {
-    this.name = name;
+  private final ArrayList<String> interestTags;
+  private final String bio;
+  private final String email;
+  private final String gitHubID;
+  private final String gitHubURL;
+  private final String image;
+  private final String location;
+  private final String name;
+
+  /**
+   * Creates a PersonObject with the given parameters
+   *
+   * @param gitHubID
+   * @param interestTags
+   * @throws IOException
+   */
+  public PersonObject(String gitHubID, ArrayList<String> interestTags) throws IOException {
     this.gitHubID = gitHubID;
-    this.description = description;
     this.interestTags = interestTags;
+
+    // GitHub query
+    final GitHub gitHub = GitHubGetter.getGitHub();
+    final GHUser userGitAccount = gitHub.getUser(gitHubID);
+    this.bio = userGitAccount.getBio();
```

Comment:
Q: I definitely think it is worth looking into what happens to these functions when the GitHub User hasn't input certain fields. ie. if the user hasn't input a bio, what will this function return?

Reply:
That makes sense, maybe just put a TODO here for now though. It seems like checking these edge cases will take a significant amount of additional work.

Reply:
Will do!